Sync-Jobs.md

```diff
@@ -1,81 +1,89 @@
 ---
 uid: Sync-Jobs
-title: Sync Jobs
+title: Download Jobs
 legacyUrl: /support/solutions/articles/44001162175-sync-jobs
 ---
 
-Sync jobs can be created from a number of areas, including, syncing by categories:
+Download jobs can be for an individual media file download or a whole TV season or show and alos possible to download a complete library. Download jobs also include those created for the [Folder Sync](Folder-Sync.md) feature.
 
-![](images/server/sync1.png)
+Download jobs are created through the context menu **Download to...** button which is available in Emby client apps for libraries and media files. This can be done using the Emby Web client app or an Emby Mobile app. For the various options available, see the [Download Options](Sync.md) article.
```

Comment:
download jobs are also created throught the "Download" button

Reply:
I am changing this paragraph to be:
`Download jobs are created through the **Download** buton within the Emby Apps that support downloads and also within context menu **Download to...** button which is available for media libraries contents. The **Download to...** button is also available in the [Emby Web client](Web-Client.md) app. For the various options available, see the [Download Options](Sync.md) article.`